paasta_tools/firewall.py

```diff
@@ -77,13 +81,19 @@ def get_rules(self, soa_dir, synapse_service_dir):
             # for several minutes after the directory disappears from soa-configs.
             return ()
 
-        if not conf.get_outbound_firewall():
+        if not conf.get_inbound_firewall() and not conf.get_outbound_firewall():
             return ()
```

Comment:
nit: Are these lines necessary anymore? You're already checking for `get_inbound_firewall` and `get_outbound_firewall` below.

Reply:
Since `tuple(list())` should indeed be equivalent, removed. :)